storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/RemotePartitionMetadataStore.java

```diff
@@ -185,4 +190,15 @@ public void maybeLoadPartition(TopicIdPartition partition) {
             topicIdPartition -> new FileBasedRemoteLogMetadataCache(topicIdPartition, partitionLogDirectory(topicIdPartition.topicPartition())));
     }
```

Comment:
nit: `maybeLoadPartition`  should have `@Override` annotation now.

Reply:
Done